### ISSUE TITLE
[Bug]: No dark mode error

### ISSUE THREAD

**Issue by makzumi** [mohamedsabil83/filament-forms-tinyeditor#94]
### What happened?

When disabling dark mode in a panel, the editor does not load since it can't find a default value for "theme", it should default to "light".

File:
**vendor/mohamedsabil83/filament-forms-tinyeditor/resources/views/tiny-editor.blade.php**
Line:
**24**

![image](https://github.com/mohamedsabil83/filament-forms-tinyeditor/assets/1649159/e294c203-0dcd-4a8d-9297-6e2c8c1e45f4)

### How to reproduce the bug

Disable dark mode and it will fail loading:

`    public function panel(Panel $panel): Panel
    {
        return $panel
            ->default()
           ...
            ->darkMode(false)
           ...
    }
`

### Package Version

2

### PHP Version

8.1

### Laravel Version

10

**Comment by makzumi** [comment]
If anyone else is facing this issue, this is my workaround:
Inject the variable inside a HtmlString.
```
TinyEditor::make('content')
    ->helperText(fn()=>new HtmlString('<script>window.theme="light"</script>'))
    ->columnSpan(1)
    ->columnSpanFull(),
```

**Comment by mohamedsabil83** [comment]
Thank you for mentioning that. Fix in [v2.0.2](https://github.com/mohamedsabil83/filament-forms-tinyeditor/releases/tag/v2.0.2)
